backend/tests/conftest.py

```
@@ -1,14 +1,15 @@
 import sys
 from pathlib import Path
 
-import pytest
-
-
 # Ensure `backend/` is on sys.path so tests can import `app.*`
 BACKEND_ROOT = Path(__file__).resolve().parents[1]
 if str(BACKEND_ROOT) not in sys.path:
     sys.path.insert(0, str(BACKEND_ROOT))
 
+import pytest
```

Ruff (0.15.2)

[error] Module level import not at top of file

(E402)